ckan/views/util.py

```diff
@@ -29,9 +29,17 @@ def primer() -> str:
     u''' Render all HTML components out onto a single page.
     This is useful for development/styling of CKAN. '''
 
-    return base.render(u'development/primer.html')
+    return base.render('development/primer.html')
+
+
+def custom_form_fields() -> str:
+    return base.render(
+        'snippets/custom_form_fields.html',
+        {'extras': [{'key': 'key', 'value': 'value'}], 'errors': {}}
+    )
 
 
 util.add_url_rule(
-    u'/util/redirect', view_func=internal_redirect, methods=(u'GET', u'POST',))
-util.add_url_rule(u'/testing/primer', view_func=primer)
+    '/util/redirect', view_func=internal_redirect, methods=('GET', 'POST',))
+util.add_url_rule('/testing/primer', view_func=primer)
+util.add_url_rule('/testing/custom_form_fields', view_func=custom_form_fields)
```

Comment:
Why do we need this endpoint? it just shows the unstyled template. Isn't it better to include `custom_form_fields.html` in the `primer.html` template so they get displayed properly?

Reply:
AFAICT it's only used by `cypress/e2e/modules/custom-fields.cy.js`

Reply:
@amercader it's just the way these tests were written. They load the site homepage then request and render a snippet into a new div in the body. For some reason.

I've tried to leave them mostly the way they were but I replaced the bit that calls `loadFixture` with code that just makes a get request and returns the content.